### PR TITLE
LTD-5430 added validation to party name

### PR DESCRIPTION
**Pull request by Tllew**
### Aim

[LTD-](https://uktrade.atlassian.net/browse/LTD-5430)

### CODE REVIEW

File: exporter/core/validators.py [hunk 1]
```diff
@@ -65,6 +65,10 @@ class PartyAddressValidator(EdifactStringValidator):
     message = "Address must only include letters, numbers, and common special characters such as hyphens, brackets and apostrophes"
 
 
+class PartyNameValidator(EdifactStringValidator):
+    message = "Party name must only include letters, numbers, and common special characters such as hyphens, brackets and apostrophes"
```
Comment (depsiatwal) [inline exporter/core/validators.py:69]: The message says no special chars but there's no reg expression for validation of this.

Comment (Tllew) [reply exporter/core/validators.py:69]: inherited from parent

Comment (depsiatwal) [reply exporter/core/validators.py:69]: ahh ok I didn't notice that. 
In which case I think we should add a few more tests for special chars even for this field.

Comment (Tllew) [reply exporter/core/validators.py:69]: I added an _ one to show it's active, we discussed previously that if the parent was tested we probably don't want to bombard all of our tests with the same test cases

**Review by saruniitr — APPROVED** [review]

**Pull request merged**